storage.js

```diff
@@ -92,13 +91,13 @@ class MemoryStorage {
 
   async saveFile(filename, data) {
     assert(!filename.includes('..'));
-    const p = path.join(__dirname, 'download', filename);
+    const p = fileURLToPath(new URL(`./download/${filename}`, import.meta.url));
```

Comment:
Can you use `fileURLToPath` once to set a `dirname` constant and simplify here and below?